src/lib/Scenes/Map/__tests__/GlobalMap-tests.tsx

```diff
@@ -27,6 +30,9 @@ describe("GlobalMap", () => {
         }
       `,
       mockData: {
+        safeAreaInsets: {
+          top: 1,
+        },
```

Comment:
This is a regular prop, not GraphQL data. What you want to do is update the MockRelayRenderer and renderRelayTree utils to the latest versions in Reaction and use the component props https://github.com/artsy/reaction/blob/2308594ff1d34da2a089b816413bed6985928445/src/DevTools/MockRelayRenderer.tsx#L25

Reply:
Super helpful, thanks!